subsys/bluetooth/host/Kconfig.l2cap

```diff
@@ -9,6 +9,7 @@ config BT_L2CAP_TX_BUF_COUNT
 	int "Number of L2CAP TX buffers"
 	default NET_BUF_TX_COUNT if NET_L2_BT
 	default BT_BUF_ACL_TX_COUNT
+	default 10 if BT_MCS
```

Comment:
@tejlmand does the conditional default have to be in an order? like before the unconditional default value?

Reply:
Moved to before the unconditional default - in the above, the value was not set to 10.